.env.production

```
@@ -0,0 +1 @@
+REACT_APP_API_URL=https://api.moodflix.store
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)